### PR TITLE
Log network location of Storybook app on startup

### LINKED ISSUE #4231: Show a URL reachable on the network when starting the Storybook server
### Support request summary

Running `start-storybook` prints a line like `info Storybook started on => http://localhost:8082/`

I'm on a Mac, and I run a Windows VM to occasionally test in IE. It would be nice if Storybook also printed a "network reachable" URL where my VM can reach it, essentially my computer's IP address:
`info Reachable on the network at http://10.30.43.152:8082`, otherwise I have to separately run `ifconfig" to figure out my current IP address and use that.

Here's where this line is currently logged:
https://github.com/storybooks/storybook/blob/master/lib/core/src/server/build-dev.js#L156

`react-styleguidist` does this, the implementation is pretty straightforward:
https://github.com/styleguidist/react-styleguidist/blob/dbb690241a6a5d273573287861bf1885e775194b/bin/styleguidist.js#L205

### Please specify which version of Storybook and optionally any affected addons that you're running

- @storybook/react 3.4.5

### Where to start

https://github.com/storybooks/storybook/blob/master/lib/core/src/server/build-dev.js#L156

https://github.com/styleguidist/react-styleguidist/blob/dbb690241a6a5d273573287861bf1885e775194b/bin/styleguidist.js#L205

### PR DESCRIPTION
**Pull request by swese44**
Issue:
Fixes #4231 

## What I did

This change logs an additional network-reachable url for the Storybook app:
screen shot 2018-09-25 at 8 51 56 pm

## How to test

Is this testable with Jest or Chromatic screenshots? Not really
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? No

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

File: lib/core/src/server/build-dev.js [hunk 1]
```diff
@@ -153,7 +154,9 @@ Would you like to run Storybook on port ${port} instead?`,
     const [stats] = await Promise.all([webpackValid, serverListening]);
     const proto = program.https ? 'https' : 'http';
     const address = `${proto}://${program.host || 'localhost'}:${port}/`;
-    logger.info(`Storybook started on => ${chalk.cyan(address)}\n`);
+    const networkAddress = `${proto}://${ip.address()}:${port}/`;
+    logger.info(`Storybook started on => ${chalk.cyan(address)}`);
```
Comment (pksunkara) [inline lib/core/src/server/build-dev.js]: Why was the `\n` removed from this line?

**Review by pksunkara — CHANGES_REQUESTED** [review]
Looks like you used yarn 1.10 to generate the lock file. Can you please use yarn 1.9?

File: lib/core/src/server/build-dev.js (hunk 1 above)
Comment (swese44) [reply lib/core/src/server/build-dev.js]: the `\n` prints an empty `info` line, removing it keeps these 2 lines adjacent in the console. note that the new network address line prints a `\n` so there's still an empty `info` line in the console after the 2 address lines

Comment (swese44) [reply lib/core/src/server/build-dev.js]: updated `yarn.lock` with yarn 1.9.4

**Review by pksunkara — APPROVED** [review]

**Comment by swese44** [comment]
thanks @pksunkara!

**Pull request merged**